src/labeler.spec.ts

```diff
@@ -0,0 +1,130 @@
+const {getFileLabel, fileStatus, labelType} = require('./labeler')
```

Comment:
If I create a folder outside `src, I get this error:

> $ ncc build src/index.ts --out dist
> ncc: Version 0.32.0
> ncc: Compiling file index.js into CJS
> ncc: Using typescript@4.5.2 (local user-provided)
> Error: [tsl] ERROR
>       TS6059: File '/home/user/src/tldr-labeler-action/test/labeler.spec.ts' is not under 'rootDir' '/home/user/src/tldr-labeler-action/src'. 'rootDir' is expected to contain all source files.
>   The file is in the program because:
>     Root file specified for compilation
>     at /home/user/src/tldr-labeler-action/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:1896789
>     at /home/user/src/tldr-labeler-action/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:37:353322
>     at _done (eval at create (/home/user/src/tldr-labeler-action/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117937), <anonymous>:7:1)
>     at eval (eval at create (/home/user/src/tldr-labeler-action/node_modules/@vercel/ncc/dist/ncc/index.js.cache.js:20:117937), <anonymous>:32:22)

Reply:
You will probably need to explicitly say what files to include in the tsconfig.json file. This error also demonstrates why we want this file outside of the ncc build, in that we do not want to risk bundling our test sources with our source code for distribution.

Reply:
The test file is not included in the bundle.